modules/dbfs/mysql.go

```diff
@@ -45,11 +45,15 @@ func (di *DatabaseImpl) getMySQLConn() (*mysqlConn, error) {
 		di.mysqldb.config.Timeout)
 	db, err := sql.Open("mysql", connString)
 	if err == nil {
-		if err = db.Ping(); err != nil {
-			di.mysqldb = nil
-			err = ErrDbNotInitialized
-		} else {
-			di.mysqldb.db = db
+		for i := uint16(1); i < di.mysqldb.config.NumRetries; i++ {
```

Comment:
lol, it was at 1 because it used to try once before doing it, but i guess it should be at 0 now